File: nodejs/7 kyu/158.js
// https://www.codewars.com/kata/57061b6fcb7293901a000ac7/train/javascript

// Passed

function headSmash(array) {
  if (typeof array === "number") {
    return "This isn't the gym!!";
  }

  let result =
    array.length !== 0
      ? array.map((string) => string.replaceAll("O", " "))
      : "Gym is empty";
  return result;
}

const output = headSmash([
  "*****************************************",
  "**  _O_   *   _O_   *   _O_   *   _O_  **",
  "** C(.)J  *  /(.)J  *  /(.)J  *  C(.)J **",
  "** _/ |_  *  _| |_  *  _( )_  *  _/ )_ *",
]);
console.log(output);

File: python/7 kyu/158.py
# https://www.codewars.com/kata/580a0347430590220e000091/train/python

# Passed

def area(d, l): 
    if d <= l:
        return "Not a rectangle"
    
    b = (d ** 2 - l ** 2) ** 0.5
    result = round(l * b, 2)
    return result

output = area(5, 4)
print(output)